so_vector/operations/default.json

```diff
@@ -1,3 +1,22 @@
+{
+  "name": "fetch_10",
+  "operation-type": "search",
+  "body": {
+    "query": {
+      "match_all": {}
+    }
+  }
+},
+{
+  "name": "fetch_1000",
+  "operation-type": "search",
+  "body": {
+    "query": {
+      "match_all": {}
```

Comment:
`match_all` will retrieve first 1000  consecutive docs based on Lucene doc IDs.  Should we instead try to fetch randomly located docs by for example using `_knn_search`?

Reply:
We use a pretty similar operation on the other places that support synthetic source. I'd *love* to have a consistent way to fetch random IDs - probably in addition to this. I'd like to be able to compare fetching hot disk blocks to fetching random ones. I know we have a random score, but I've a bit worried the random score will dominate the time.